lib/plugins/aws/index.js

```diff
@@ -172,14 +172,18 @@ class SDK {
   getServerlessDeploymentBucketName(stage, region) {
     const stackName = `${this.serverless.service.service}-${stage}`;
     return this.request('CloudFormation',
-      'describeStackResource',
+      'describeStacks',
```

Comment:
@eahefnawy I found the corresponding PR where we've moved away from `describeStacks` to `describeStackResource`: https://github.com/serverless/serverless/pull/1639